crates/tabby-common/assets/languages.toml

```diff
@@ -212,3 +212,34 @@ top_level_keywords = [
     "function",
     "type",
 ]
+
+[[config]]
+languages = ["css"]
+line_comment = "/*"
```

Comment:
this won't work given how we construct line comments string for building context into prompt... (`line_comment + code`)

🤔

Reply:
I'm sorry about that, I haven't looked into the implementation. I will try to fix it.

Reply:
No worries - this is indeed a flaw of context building implementation. Thanks for providing such a case to initiate the improvements